src/main/java/com/example/simulator/utils/forecast/WeatherForecast.java

```diff
@@ -0,0 +1,44 @@
+package com.example.simulator.utils.forecast;
+
+import com.example.simulator.utils.forecast.data.CurrentWeather;
+import com.example.simulator.utils.forecast.data.WeatherData;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonMappingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.gson.FieldNamingPolicy;
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.mashape.unirest.http.HttpResponse;
+import com.mashape.unirest.http.JsonNode;
+import com.mashape.unirest.http.Unirest;
+import com.mashape.unirest.http.exceptions.UnirestException;
+
+public class WeatherForecast {
+
+    public static final String urlForecast = "https://api.open-meteo.com/v1/forecast";
+
+    public static final Double latitude = 53.1221;
```

Comment:
Może współrzędne geograficzne powinny być przechowywane dla każdej szklarni oddzielnie? Z racji na to, że teoretycznie mogą być w zupełnie różnych miejscach, a więc i temperatura zewnętrzna może się różnić.